native/CMakeLists.txt

```diff
@@ -2616,57 +2616,57 @@ function(compile_ts input_file_ts output_file)
     )
 endfunction()
 
-compile_ts(${CWD}/../cocos/core/builtin/effects.ts ${CWD}/cocos/core/builtin/effects.js)
-compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl1.ts ${CWD}/cocos/core/builtin/shader-sources/glsl1.js)
-compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl3.ts ${CWD}/cocos/core/builtin/shader-sources/glsl3.js)
-compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl4.ts ${CWD}/cocos/core/builtin/shader-sources/glsl4.js)
+# compile_ts(${CWD}/../cocos/core/builtin/effects.ts ${CWD}/cocos/core/builtin/effects.js)
+# compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl1.ts ${CWD}/cocos/core/builtin/shader-sources/glsl1.js)
+# compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl3.ts ${CWD}/cocos/core/builtin/shader-sources/glsl3.js)
+# compile_ts(${CWD}/../cocos/core/builtin/shader-sources/glsl4.ts ${CWD}/cocos/core/builtin/shader-sources/glsl4.js)
 
 set(CC_GENERATED_FILES
-    ${CWD}/cocos/core/builtin/Effects.cpp
+    # ${CWD}/cocos/core/builtin/Effects.cpp
     ${CWD}/cocos/core/builtin/DebugInfos.cpp
-    ${CWD}/cocos/core/builtin/shader-sources/glsl1.cpp
-    ${CWD}/cocos/core/builtin/shader-sources/glsl3.cpp
-    ${CWD}/cocos/core/builtin/shader-sources/glsl4.cpp
+    # ${CWD}/cocos/core/builtin/shader-sources/glsl1.cpp
+    # ${CWD}/cocos/core/builtin/shader-sources/glsl3.cpp
+    # ${CWD}/cocos/core/builtin/shader-sources/glsl4.cpp
 )
 
 add_custom_target(builtin-res
```

Comment:
Could remove `builtin-res` task now.

Reply:
But builtin-res still be used to generate DebugInfo, right? 
<img width="690" alt="image" src="https://user-images.githubusercontent.com/15109777/177027601-73c47a11-12ab-4201-a33f-2c69cb09932a.png">